src/components/icons/CheckboxChecked.tsx

```diff
@@ -13,6 +13,7 @@ export default function CheckboxCheckedIcon(props: CheckboxCheckedIconProps) {
       height="16"
       aria-hidden="true"
       viewBox="-4 -4 39 39"
+      className="text-grey-1"
```

Comment:
This class introduces an undesired change on the icon when used standalone.

![image](https://github.com/hypothesis/frontend-shared/assets/2719332/2e7d88c9-8e1f-4077-874b-ab9cbd52d4b7)

However, I can remove this class and the icon still has the new look-and-feel when used in the `Checkbox` component.

Can you clarify the context of this change, just to make sure I'm not missing anything?

```suggestion
```

Reply:
@acelaya I removed this, originally I wanted to ensure the color of the checkbox would be consistent (i.e. dark gray) instead of taking whatever the parent's color was (i.e. `currentColor`), but I think it's preferable to be flexible.